sydent/http/servlets/store_invite_servlet.py

```diff
@@ -138,6 +139,9 @@ def render_POST(self, request: Request) -> JsonDict:
 
         for keyword in self.sydent.config.email.third_party_invite_keyword_blocklist:
             for (key, value) in args.items():
+                # make sure the blocklist doesn't stomp on invite_client_location url
+                if key == "org.matrix.web_client_location":
+                    value = re.sub(r"^(https?://)", "", value)
```

Comment:
It may be possible to simplify the regex by removing the parentheses.
```suggestion
                    value = re.sub(r"^https?://", "", value)
```